apps/onedev/11.6.5/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   onedev:
-    image: 1dev/server:11.6.5
+    image: 1dev/server:11.6.6
     container_name: ${CONTAINER_NAME}
     restart: always
     networks:
```

Comment:
There is a single change in the Docker image version from `1.dev/server:11.6.5` to `1.dev/server:11.6.6`. This update suggests that there was an upgrade or patch release of the `onedev/server`, which might affect functionality or performance. No other irregularities were found.